ipfx/ephys_data_set.py

```diff
@@ -115,20 +114,22 @@ def sweep(self, sweep_number):
         sweep_meta_data = self.get_sweep_meta_data(sweep_number)
         sampling_rate = sweep_data['sampling_rate']
         dt = 1. / sampling_rate
-        sweep_start_ix, sweep_end_ix = ep.get_sweep_epoch(sweep_data['response'])
+        t = np.arange(0, len(sweep_data['stimulus'])) * dt
+        assert len(sweep_data['stimulus']) == len(sweep_data['response'])
 
-        response = sweep_data['response'][sweep_start_ix:sweep_end_ix+1]
-        stimulus = sweep_data['stimulus'][sweep_start_ix:sweep_end_ix+1]
-        t = np.arange(sweep_start_ix, sweep_end_ix + 1) * dt
+        try:
+            epochs = sweep_data['epochs']
+        except KeyError:
+            epochs = {}
```

Comment:
I think you should be able to replace 120-123 with just `epochs = sweep_data.get('epochs', dict)`

Reply:
good idea